src/Util/Caller.js

```diff
@@ -1,13 +1,14 @@
 
 
-const url_backend ="http://localhost:8080"
+const url_backend ="https://park-inn-ispp-be.herokuapp.com/"
+const url_frontend = "https://park-inn-ispp-fe.herokuapp.com/"
 
 
      async function call(path_to_call,method,body){
 
         const requestOptions = {
             method: method,
-            headers: { 'Content-Type': 'application/json', 'Access-Control-Allow-Origin' : 'https://park-inn-ispp-fe.herokuapp.com', "mode": "cors"},
+            headers: { 'Content-Type': 'application/json', 'Access-Control-Allow-Origin' : url_frontend, "mode": "cors"},
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/park-inn-ispp/park-inn-frontend/pullRequest?prid=9143605)
```suggestion
            headers: { "Content-Type": 'application/json', 'Access-Control-Allow-Origin' : url_frontend, "mode": "cors"},
```